build-parent/pom.xml

```diff
@@ -32,7 +32,7 @@
         <!-- These properties are needed in order for them to be resolvable by the documentation -->
         <!-- The Graal version we suggest using in documentation - as that's
            what we work with by self downloading it: -->
-        <graal-sdk.version-for-documentation>19.3.1</graal-sdk.version-for-documentation>
+        <graal-sdk.version-for-documentation>20.0.0</graal-sdk.version-for-documentation>
```

Comment:
I am not sure we upgraded to GraalVM 20 already

Reply:
This PR https://github.com/quarkusio/quarkus/pull/7427 updated the version to 20.0.0.

Reply:
Right, the image is there, but we need #7320 and #7321 to be merged first to officially say we support GraalVM 20 :wink:

Reply:
We need to go back to 19.3.1 here.

Reply:
Version reverted back to 19.3.1